### PR TITLE
{tools}[foss/2022a] jax v0.3.23 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0105u36b.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/71371c6ffc1f24bc4e3a4098d43cc7c7 for a full test report.

File: easybuild/easyconfigs/j/jax/jax-0.3.23-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        'patches': [
+            'jax-0.3.9_relax-test-tolerance.patch',
+        ],
+        'source_tmpl': '%(name)s-v%(version)s.tar.gz',
+        'source_urls': ['https://github.com/google/jax/archive/'],
+        'checksums': [
+            {'jax-v0.3.23.tar.gz': 'fa8c68a82fa2fcf3d272bf239c77e7028bb6077466a53349ce85f6e85ed623db'},
+            {'jax-0.3.9_relax-test-tolerance.patch':
+             '3da3c8b4d9ff3449b51a4f39d6bbadd348ea3bd4ca493a6f1292743f86fa7b3d'},
+        ],
```
[the remark this answers is not in the document]
Comment (branfosj) [reply easybuild/easyconfigs/j/jax/jax-0.3.23-foss-2022a.eb:103]: For the first two https://github.com/boegel/easybuild-easyconfigs/pull/87

Which also has a sync with develop in there

[the remark this answers is not in the document]
Comment (branfosj) [reply easybuild/easyconfigs/j/jax/jax-0.3.23-foss-2022a.eb:103]: With https://github.com/boegel/easybuild-easyconfigs/pull/87 this builds and the tests pass.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
@boegelbot please test @ jsc-zen2

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0210u13b.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) CPU E5-2640 v4 @ 2.40GHz (broadwell), Python 3.6.8
See https://gist.github.com/06462bf4a0745b9a1e61558ba73a5a6a for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3101.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/dd53f518e7ad6a1e98f0a56ad2622bd3 for a full test report.

**Pull request merged**